modules/samples/watertank/component/ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.watertank.device-view/device-view.hbs

```diff
@@ -0,0 +1,68 @@
+{{#zone "topCss"}}
```

Comment:
This is a handle bar template in UUF 1.5 we are not adding license headers as it is overhead to template rendering.